docs/default-locale-timezone.adoc

```diff
@@ -0,0 +1,93 @@
+:page-title: @DefaultLocale and @DefaultTimeZone
+:page-description: JUnit Jupiter extensions to change the values returned from Locale.getDefault() and TimeZone.getDefault()
+
+The `@DefaultLocale` and `@DefaultTimeZone` annotations can be used to change the values returned from `Locale.getDefault()` and `TimeZone.getDefault()`.
+Both annotations work on the test class level and on the test method level.
+After the annotated element has been executed the initial default value is restored.
+
+== `@DefaultLocale`
+
+The default `Locale` is specified according to the constructors of https://docs.oracle.com/javase/10/docs/api/java/util/Locale.html[Locale].
+
+[source,java]
+----
+@Test
+@DefaultLocale(language = "en")
+void test_with_lanuage() {
+    assertEquals(new Locale("en"), Locale.getDefault());
+}
+
+@Test
+@DefaultLocale(language = "en", country = "EN")
+void test_with_lanuage_and_country() {
+    assertEquals(new Locale("en", "EN"), Locale.getDefault());
+}
+
+@Test
+@DefaultLocale(language = "en", country = "EN", variant = "gb")
+void test_with_lanuage_and_country_and_vairant() {
+    assertEquals(new Locale("en", "EN", "gb"), Locale.getDefault());
+}
+----
+
+Note that variant can only be specified when the country is also specified.
+If a variant is specified without a country an `ExtentionConfigurationException` will be thrown.
```

Comment:
If `variant` is specified without `country`, an `ExtentionConfigurationException` will be thrown.